src/modules/editor/SoundEditor/MusicManager.ts

```diff
@@ -0,0 +1,44 @@
+
+import * as Tone from "tone";
+import { Time } from "tone/build/esm/core/type/Units";
```

Comment:
I don't think you should include from a built directory -- if possible, install the package and use it instead :]